### PR TITLE
Fix Assimp 16-bits and 32-bits textures

### PR DESCRIPTION
**Pull request by Meakk**
Fix https://github.com/f3d-app/f3d/issues/586

### CODE REVIEW

**Comment by Meakk** [comment]
@bisechen Can I add your dataset in the test database? Are you the original author?

**Comment by bisechen** [comment]
> 我可以在測試資料庫中添加您的數據集嗎？你是原作者嗎？

File: testing/data/DATA_LICENSES.md [hunk 1]
```diff
@@ -39,6 +39,7 @@
 - vase_4comp.vti: VTK Data: BSD-3-Clause
 - WaterBottle.glb: glTF-Sample-Models: Public Domain
 - world*: VTK Data: BSD-3-Clause
+- 16bits.*: @bisechen: Public Domain
```
Comment (Meakk) [inline testing/data/DATA_LICENSES.md:42]: @bisechen I've put a "Public Domain" license. Let me know if it works for you or if you prefer another license.

Comment (mwestphal) [reply testing/data/DATA_LICENSES.md:42]: CC0 would be better than public domain if choice is possible

File: plugins/assimp/module/vtkF3DAssimpImporter.cxx [hunk 2]
```diff
@@ -220,6 +220,7 @@ class vtkF3DAssimpImporter::vtkInternals
 
     vTexture->MipmapOn();
     vTexture->InterpolateOn();
+    vTexture->SetColorModeToDirectScalars();
```
Comment (mwestphal) [inline plugins/assimp/module/vtkF3DAssimpImporter.cxx:223]: I wonder how this can work at all without this line.

Comment (Meakk) [reply plugins/assimp/module/vtkF3DAssimpImporter.cxx:223]: Direct scalar is the default for 8-bits textures (which is the most common format)

File: testing/data/DATA_LICENSES.md (hunk 1 above)
Comment (Meakk) [reply testing/data/DATA_LICENSES.md:42]: Ok. Please @bisechen let us know if you are ok with cc0

File: plugins/assimp/module/vtkF3DAssimpImporter.cxx (hunk 2 above)
Comment (mwestphal) [reply plugins/assimp/module/vtkF3DAssimpImporter.cxx:223]: ok

**Review by mwestphal — APPROVED** [review]

File: testing/data/DATA_LICENSES.md (hunk 1 above)
Comment (bisechen) [reply testing/data/DATA_LICENSES.md:42]: > 還行。如果您對cc0沒問題，請告訴我們

This is only a temporary use to test the 16 and 32 bit color depth file of F3D 2.0, not a "work", so CC0 is not a problem. ^^

Comment (Meakk) [reply testing/data/DATA_LICENSES.md:42]: We are trying to be respectful of other people's work, even if it's a quick test model. Thank you for accepting the CC0 license :)

**Pull request merged**